src/create-model.jl

```diff
@@ -362,6 +422,16 @@ function create_model(graph, representative_periods, dataframes, timeframe; writ
         end
     end
 
+    sub_df_is_charging_binary = filter(
+        row -> row[:asset] in [a for a in Asb if graph[a].use_binary_storage_method == :binary],
+        df_is_charging;
+        view = true,
+    )
```

Comment:
I think a valid solution here is to append the value of `graph[a].use_binary_storage_method` to `df_is_charging` - I am reading this out of order, so I might have commented in another file already when you read this.
The idea here is to create a new column in `df_is_charging` then change to something like

```Julia
sub_df_is_charging_binary = subset(
  df_is_charging,
  :asset => ByRow(in(Asb)),
  :use_binary_storage_method => ByRow(==(:binary)),
  view=true
)
```

This should be somewhat more efficient.